src/test/java/rv/atqc/gttesting/editmyprofiledata/po/US_EditMyProfileDataWithChrome.java

```diff
@@ -0,0 +1,154 @@
+package rv.atqc.gttesting.editmyprofiledata.po;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import io.github.bonigarcia.wdm.ChromeDriverManager;
+import rv.atqc.gttesting.archex.EditProfilePage;
+import rv.atqc.gttesting.archex.HeadForGuestUserPage;
+import rv.atqc.gttesting.archex.ProfilePage;
+import rv.atqc.gttesting.res.Resources;
+
+public class US_EditMyProfileDataWithChrome {
+
+	private WebDriver driver;
+	private ProfilePage pageProfile;
+	private EditProfilePage page;
+
+	@BeforeMethod
+	public void before() {
+		ChromeDriverManager.getInstance().setup();
+		driver = new ChromeDriver();
+		driver.get(" http://green-tourism.herokuapp.com");
+		driver.manage().window().maximize();
+		pageProfile = new HeadForGuestUserPage(driver).clickLeftNavButton().clickLogIn()
+				.setLoginField(Resources.LogInMessage.USER_EMAIL).setPasswordField(Resources.LogInMessage.USER_PASSWORD).clickLogInButton().clickLeftNavButton()
+				.clickProfile();
+		page = pageProfile.clickEdit();
+	}
+
+	@Test
+	public void verifyNameOfEditButton() {
+		Assert.assertTrue(pageProfile.getEditButton().getText().contains("Edit"));
```

Comment:
I just moved written test to page object, and tried minimised change of logic. It was assertTrue before me it was at demo, so i thought that author had some extra reason for this.